test/regression/2015-08-05-invalid-fork.c

```diff
@@ -0,0 +1,24 @@
+/* Reported by @kren1 in #262 
+   The test makes sure that the string "Should be printed once" 
+   is printed a single time. 
+*/
+
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out %t.bc >%t1.log
+// RUN: grep "Should be printed once"  %t1.log
```

Comment:
@delcypher I agree that `FileCheck` is much better than `grep` and we should try to enforce it. However, in some cases `FileGrep` might be too verbose. For example, consider the case in which you want to check for the same pattern multiple time (we have multiple cases in our test suite). In such a case, I would actually use `grep`.